monitoring/rid_qualifier/README.md

```diff
@@ -0,0 +1,11 @@
+# RID Qualifier Tests
+
+This directory contains a series of tests for qualifying Network Remote ID compliance. It will contain four things:
+
+1. **Flight track dataset generator** (`flight_track_factory.py`): An API to generate multiple flight paths and patterns within a specified bounding box. You can specify a bounding box in any part of the world the generator will create a grid for flights for the bounds provided. In addition, circular flight paths will be generated within that grid. The flights will follow a circular path and these are generated in the form of latitude / longitude co-ordinates within the bounding box.
+
+2. **Test payload generator** (`aircraft_state_replayer.py`): Once the flight tracks are generated, we simulate flights and "fly" them emitting positions to be fed to the test harness, a file is generated that can be submitted by the test harness to the API, this way the aircraft state can be replayed. Specific details metadata about the flights are generated and added in preparation for submission to the API.

```

Comment:
Thank you for your comments. Yes I agree with your broad comments. I do think that "simulator" is not the correct term for the state re-player, since when I think of simulation, I think of the SITL and more sophisticated / mature software, this is meant for a much different use-case, so anything that avoids the name is fine with me. I have updated the changes to the README